components/raftstore/src/store/peer.rs

```diff
@@ -2322,6 +2334,17 @@ where
                             "region_id" => self.region_id,
                         );
                     }
+                    // After the leadership changed, send `CasualMessage::Campaign` to the target
+                    // peer to campaign leader if there exists uncampaigned regions. It's used to
+                    // ensure there only one valid peer apply for campaign in the newly created raft
+                    // group to prevent unexpected behaviors in rare scenarios (e.g. #12410 and
+                    // #17602.),
```

Comment:
Would this be better?
```suggestion
                    // ensure that a leader is elected promptly for the newly created Raft 
                    // group, minimizing availability impact (e.g. #12410 and #17602.).
```